### DISCUSSION TITLE
[4.6] OCI AWS S3 link broken for  the job Provision/DataPipeline/InfluxDB

### DISCUSSION THREAD

**Discussion by subhashchandrab** [Sunbird-Ed/Community#618] [Installation] [answered]
I'm getting the below error in Provision/DataPipeline/InfluxDB

```
TASK [influxdb : Download InfluxDB-Relay .deb package from S3] *****************
fatal: [10.*.*.*]: FAILED! => {
    "changed": false,
    "dest": "/tmp/influxdb-relay.deb",
    "elapsed": 1,
    "response": "HTTP Error 404: Not Found",
    "status_code": 404,
    "url": "https://s3.ap-south-1.amazonaws.com/ekstep-third-party-libraries/influxdb-relay/influxdb-relay_535181e_amd64.deb"
}

MSG:

Request failed
```

Looks like the aws s3 link https://s3.ap-south-1.amazonaws.com/ekstep-third-party-libraries/influxdb-relay/influxdb-relay_535181e_amd64.deb is broken. Can you please check and confirm any other alternative link so that the job can proceed.

**Comment by ashokreddy1208** [comment]
@gandham-santhosh @santhosh-tg Can you please look into this

**Reply by gandham-santhosh** [reply]
@subhashchandrab can you please use this https://dl.influxdata.com/influxdb/releases/influxdb_1.8.3_amd64.deb and check

**Reply by ddevadat** [reply]
on using the above link it downloaded..but failed at install

```
TASK [influxdb : Install influxdb_relay_deb_package_name] **********************
fatal: [10.90.8.6]: FAILED! => {
    "changed": false
}

STDOUT:

(Reading database ... 207809 files and directories currently installed.)
Preparing to unpack /tmp/influxdb-relay.deb ...
Unpacking influxdb (1.8.3-1) over (1.1.1+dfsg1-4+deb9u1ubuntu1) ...
Processing triggers for man-db (2.8.3-2ubuntu0.1) ...



STDERR:

dpkg: error processing archive /tmp/influxdb-relay.deb (--install):
 trying to overwrite '/usr/bin/influx', which is also in package influxdb-client 1.1.1+dfsg1-4+deb9u1ubuntu1
dpkg-deb: error: paste subprocess was killed by signal (Broken pipe)
Synchronizing state of influxdb.service with SysV service script with /lib/systemd/systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install disable influxdb
Removed /etc/systemd/system/influxd.service.
/usr/bin/deb-systemd-helper: error: unable to read influxdb.service
Errors were encountered while processing:
 /tmp/influxdb-relay.deb



MSG:

dpkg --force-confdef --force-confold -i /tmp/influxdb-relay.deb failed
```

**Reply by heungheung** [reply]
@gandham-santhosh it is different. The one you provided is influxdb. What is missing here is influxdb-relay. I was trying to compile the relay from source but that didn't work (using the same commit hash)
https://github.com/influxdata/influxdb-relay/tree/535181e7b0e0b291ae576868325c4cce4c096c37
fpm cannot be install, that one is still ok as I was able to force the fpm to 0.x version. fpm got installed. Now, I am hitting not able to 
```
go get github.com/sparrc/gdm
```
coz package "golang.org/x/tools/go/vcs" is missing. Go 1.6.2 is really really old..... 

If anyone has a copy of influxdb-relay .deb, please kindly share.

**Reply by gandham-santhosh** [reply]
@anandp504 mentioned that influxdb is not required now as the metrics are available in Prometheus now. Are you using Influx for any other purpose? We will be updating the Ed installation documentation

**Reply by heungheung** [reply, marked as answer]
working on a 4.x installation. :-(  Anyway, I finally go one from my old old old environment. 

Just in case anyone need it (most likely, you shouldn't), ping me offline. 

Closing this issue

**Reply by heungheung** [reply]
ooops... need @subhashchandrab to mark it as answer. Please Subhash. It is fixed.
